pkg/tekton/syntax/constants.go

```diff
@@ -14,5 +14,5 @@ const (
 	KanikoDockerImage = "gcr.io/kaniko-project/executor:9912ccbf8d22bbafbf971124600fbb0b13b9cbd6"
 
 	// DefaultContainerImage - the default image used for pipelines if none is specified.
-	DefaultContainerImage = "gcr.io/jenkinsxio/builder-maven"
+	DefaultContainerImage = "maven"
```

Comment:
I think in the next step we should switch from `maven` to `jx`. There is no conceivable reason why we would need a Java-based image.

Reply:
the main reason historically is we use maven for updating the version numbers in maven project releases. For all other languages we can modify the code (e.g. package.json). I guess the meta pipeline can add the maven version in as a separate step?